azure-monitor-opentelemetry/tests/diagnostics/test_status_logger.py

```diff
@@ -234,31 +234,31 @@ def test_disabled_log_status_failed_initialization(self, mock_getpid):
         check_file_is_empty()
 
     @patch(
-        "azure.monitor.opentelemetry.distro._diagnostics._status_logger._STATUS_LOG_PATH",
+        "azure.monitor.opentelemetry.diagnostics._status_logger._STATUS_LOG_PATH",
```

Comment:
Is the azure sdk style to leave the module folder without _ but put _ in the filenames themselves? OTel, for instance uses _logs as a folder name.

Reply:
Same goes for the autoinstrumentation folder

Reply:
Yes it's following azure-sdk styling.